content/en/blog/2019/dns-cert/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Istio DNS certificate management
+description: Provision and manage DNS certificates in Istio.
+publishdate: 2019-10-29
+attribution: Lei Tang (Google)
+keywords: [security, kubernetes, certificates, DNS]
+target_release: 1.4
+---
+
+The DNS certificates used by Istio control plane components (e.g., Galley, Sidecar Injector)
+are by default managed by Citadel, which is a large component, maintains its own private
+signing key, and also acts as a CA.
+
+In Istio 1.4, we introduce a new feature to securely provision and manage DNS certificates
+signed by Kubernetes CA in Istio, which has the following advantages.
+
+* This feature removes the dependency on Citadel for DNS certificates and is
+more lightweight than Citadel.
+
+* Unlike Citadel, this feature doesn't require maintaining a private signing key, which enhances security.
+
+* This feature simplifies root certificate distribution to TLS clients
+as there is no waiting for Citadel to generate and distribute its CA cert.
+
+The following diagram shows the architecture of provisioning and managing DNS certificates in Istio.
+
+{{< image width="50%"
```

Comment:
We removed the name Chiron from the text so now the diagram needs it removed also, otherwise readers won't know what Chiron means in the diagram. Is there a way to describe this new features behavior without referring to the internal name of it?

Everything else looks great, thanks!!!

Reply:
The architecture needs to have concrete components (i.e., Galley, Sidecar Injector, Chiron). Otherwise, it will be too vague and difficult for readers to understand. I have added one sentence to explain the Chiron component.